### DISCUSSION TITLE
Soroban bet contract

### DISCUSSION THREAD

**Discussion by Nodecafe-dev** [stellar/sorobanathon#33] [General]
Hi, 
I've written a bet contract using soroban. This is to be used to create bets on sport events. 

repo :https://github.com/Nodecafe-dev/soroban-bet-contract

Cheers,

Jino

**Comment by achallagundla** [comment]
Heya @Nodecafe-dev! 
Thank you for your submission! Will get back to you shortly after a review!

**Comment by achallagundla** [comment]
Heya @Nodecafe-dev ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Great piece BTW! Can't wait to see you keep tinkering with Soroban and documenting your finding!

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya
